tests/Core/Slider/FluentSliderTests.razor

```diff
@@ -6,7 +6,7 @@
     public void FluentSlider_HorizontalImplicit()
     {
         // Arrange && Act
-        var cut = Render(@<FluentSlider Min="0" Max="100" Step="10"></FluentSlider>);
+        var cut = Render(@<FluentSlider Min="0" Max="100" Step="10" Label="label"></FluentSlider>);
```

Comment:
Could you avoid updating existing tests to check for new features?
It would be better to create new tests to validate the `Label` attribute.

Reply:
Tests are in order now